index.html

```diff
@@ -0,0 +1,115 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile First Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <div class="placeholder">
+            <img src="images/Logo Placeholder.png" alt="My Logo">
+        </div>
+        
+        <div class="hamburger">
+            <a href="#">
+                <img src="images/hamburger.png" alt="Hamburger Icon">
+            </a>
+        </div>
+    </header>
+
+    <section class="headline">
+        <div class="primarytext">
+            <h1>Hey There. <br>I'm Ezra.</h1>
+        </div>
+
+        <div class="supporting">
+            <h2>I'm a Software Developer</h2>
+        </div>
+
+        <div class="paragraph">
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.</p>
+        </div>
+
+        <div class="social">
+            <div>
+                <a href="#">
+                    <img src="images/github.png" alt="Github">
+                </a>
+            </div>
+            <div>
+                <a href="#">
+                    <img src="images/linkedin.png" alt="LinkedIn">
+                </a>
+            </div>
+            <div>
+                <a href="#">
+                    <img src="images/thumbs.png" alt="Thumbs">
+                </a>
+            </div>
+            <div>
+                <a href="#">
+                    <img src="images/twitter.png" alt="Twitter">
+                </a>
+            </div>
+            <div>
+                <a href="#">
+                    <img src="images/M.png" alt="M">
+                </a>
+            </div>
+        </div>
```

Comment:
- [ ] Your social media icons look fantastic! :100: :+1: However, for better semantics, it is recommended to group the icons using an unordered list (`ul`) and add a list item (`li`) for each icon. Additionally, to make the icons clickable, each list item should contain a link (`a` tag) to the corresponding social media profile (You have already done that). This approach not only helps to improve the structure of the code but also enhances the accessibility of the website, allowing users to navigate social media profiles more efficiently. Moreover, using a `ul` tag will ensure that the icons are treated as a group and, if desired, can be styled as such :+1:.